examples/kotlin/hello_world/MainActivity.kt

```diff
@@ -33,10 +34,9 @@ class MainActivity : Activity() {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_main)
 
-    Envoy.load(baseContext)
-
     // Create Envoy instance with config.
-    envoy = Envoy(baseContext, loadEnvoyConfig(baseContext, R.raw.config))
+    val envoyEngine = AndroidEngineImpl(baseContext)
+    envoy = Envoy(envoyEngine, loadEnvoyConfig(baseContext, R.raw.config))
```

Comment:
In some ideal future, the public interface will only require a single line here (but I think we're all on the same page on this).

Reply:
@rebello95 and I were thinking about having a builder class for creating an Envoy instance so we don't have to do this. It'll bundle together the configuration and log levels also.

Reply:
iOS example: https://github.com/lyft/envoy-mobile/pull/343